src/@types/Language.ts

```diff
@@ -1,8 +1,18 @@
+// -- start tsdoc imports --
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import type { LoadingScene } from "#app/loading-scene";
+// -- end tsdoc imports --
```

Comment:
NIT: To make it easier to add more in the future use multi-line disable/enable
```suggestion
/* eslint-disable @typescript-eslint/no-unused-vars */
import type { LoadingScene } from "#app/loading-scene";
/* eslint-enable @typescript-eslint/no-unused-vars */
// -- end tsdoc imports --
```